templates/trailer.tmpl

```diff
@@ -1,5 +1,5 @@
       </div>
     </div>
-    <div class="footer">CUPS and the CUPS logo are trademarks of <a href="http://www.apple.com">Apple Inc.</a> Copyright &copy; 2007-2019 Apple Inc. All rights reserved.</div>
+    <div class="footer">Copyright © 2021-2022 OpenPrinting. All rights reserved.</div>
```

Comment:
@Rishabh-792 

Please use the html tag for copyright, not actual character '©' - by tag we can ensure it is shown properly everywhere.

Reply:
Done !

Thanks @zdohnal .